console/Purge.php

```diff
@@ -0,0 +1,22 @@
+<?php namespace SureSoftware\Maillog\Console;
+
+use Illuminate\Console\Command;
+use SureSoftware\MailLog\Models\MailLog;
+use SureSoftware\MailLog\Models\Settings;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputOption;
+
+class Purge extends Command
+{
+    protected $name = 'maillog:purge';
```

Comment:
Given the PR is named "purge after x days" to mean indicates that it should have a parameter/flag in the command to indicate how many days you want to purge after, which overrides the setting stored in the DB. Can you please update this?